sample_tracking/app/edit/page.tsx

```diff
@@ -117,6 +117,7 @@ export default function Edit() {
         const sampleData = {
             ...updatedFormData,
             visibility: 'private',
+            d18O_cel: updatedFormData.d18O_cel ? updatedFormData.d18O_cel.map((value: string) => parseFloat(value)) : [],
```

Comment:
did this ever work without d18O_cel?  Was this an oversight or did we t-test on a different column? (O_wood, but that does not all sound right to me)

Reply:
I believe this was an oversight from our end.